### PR TITLE
Update UI 

### LINKED ISSUE #7: Split Home description in CharacterCard to into different line
When Planet name or population number is big, cards is look messy

### LINKED ISSUE #29: Create Header component for movies/movie page

### PR DESCRIPTION
**Pull request by RayRedGoose**
**This PR contains next changes:**
- Added Page404 component for incorrect URLs

page404

- Added BB-8 animation on mouse moves to MoviesPage

bb-animation

- Updated styles for CharacterCard
- Added title for CharacterBox and button with favorites counter going to FavoritesPage

characters-box

- Added styles for FavoritesPage 

fav-page

- Added minor changes to others existing components

**This PR resolves next issues:**

resolves #29
resolves #7

### CODE REVIEW

File: src/components/BBAnimated/BBAnimated.js [hunk 1]
```diff
@@ -0,0 +1,117 @@
+import React, { Component } from 'react'
+import './BBAnimated.scss'
+
```
Comment (spaceplesiosaur) [inline src/components/BBAnimated/BBAnimated.js:3]: I'm definitely going to credit the author in the ReadMe, but would be be ok to add some commented out code crediting the author for this below the imports?  It's what I did on my last project and Brittany seemed to think it was enough

Comment (spaceplesiosaur) [reply src/components/BBAnimated/BBAnimated.js:3]: Nevermind!  I see it in the scss file.

**Review by spaceplesiosaur — COMMENTED** [review]
When I pull down I'm still not seeing styling on the UserProfile - is this supposed to be included?  I do see it in your screenshot.
Screen Shot 2019-12-07 at 4 30 59 PM

I'm also seeing that the character files look different from your screenshots: 
Screen Shot 2019-12-07 at 4 32 55 PM

based on our earlier convo I'm guessing this is because I'm just pulling down the UI branch and not the functionality branch, but I wanted to check and make sure.

How do I get the 404 to show up?  it's currently not showing up when I put in a bad URL.  Lmk if that's functionality stuff too, I'm about to check that branch!

**Comment by RayRedGoose** [comment]
All css styles based on other branch.  There was a small changes in component and I missed them to push it here.

File: src/components/BBAnimated/BBAnimated.js (hunk 1 above)
Comment (RayRedGoose) [reply src/components/BBAnimated/BBAnimated.js:3]: Yes, in scss was first author already, and I just added author who created it for React

**Pull request merged**